docker/hooks/build

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# navigate from Dockerfile folder to project root
+cd ..
+
+### upgrade to Docker 18 ###
```

Comment:
Do we want to predicate this hackishness on something we can detect from the environment? We're running on some opaque vm here that we have on control over or insight into and that might change at any point, right?

Reply:
What exactly do you mean with "something we can detect from the environment"?

I have no idea why DockerHub runs Docker 17, maybe because of some LTS strategy...

I mean we could apply syncthing for admission to the [Library's official images](https://github.com/docker-library/official-images). Then syncthing would be built on dedicated machines without Qemu-hacks, but at the cost of quick releases. Afaik one needs to send a PR to update an official image, but I could be wrong about that.